lib/checks/mobile/target-offset.json

```diff
@@ -0,0 +1,14 @@
+{
+  "id": "target-offset",
+  "evaluate": "target-offset-evaluate",
+  "options": {
+    "minOffset": 24
+  },
+  "metadata": {
+    "impact": "serious",
+    "messages": {
+      "pass": "Target is sufficiently offset from its closest neighbor (${data.closestOffset}px should be at least ${data.minOffset}px)",
```

Comment:
I put it there because I figured if someone is actually reading these, they're probably trying to figure out why axe-core is passing something they thought should fail. We can do that without putting it in the pass message though. Let me know if you want this removed. It'll still be in `.data`, even if it's not in the message. I can go either way.